packages/repository/src/model.ts

```diff
@@ -214,7 +214,7 @@ function asObject(value: any, options?: Options): any {
 /**
  * Base class for models
  */
-export abstract class Model {
```

Comment:
Any reason it's no longer abstract? IIUC, the `Model`, `Entity` and `ValueObject` classes should only be extended from and not be used directly.

Reply:
TypeScript does not allow using `Model` as a value (class) if it's abstract. I can revert these changes and only test if the class is decorated with `@model`.